<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aide</class>
 <widget class="QDialog" name="aide">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aide</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>471</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Aide</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>431</width>
      <height>371</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; font-style:italic; text-decoration: underline;&quot;&gt;bonjour et bienvenu &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;dans notre application&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Aziz ben ismail&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Yassin hkiri&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Haythem&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;lina &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Lamis &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Shayma&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>750</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour au menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Q">
   <property name="geometry">
    <rect>
     <x>994</x>
     <y>760</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ph">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>451</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>0.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="Digital_clock">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_p">
   <property name="geometry">
    <rect>
     <x>-220</x>
     <y>-80</y>
     <width>1421</width>
     <height>941</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_note">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>730</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Note</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cal">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>730</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculatrise</string>
   </property>
  </widget>
  <zorder>label_p</zorder>
  <zorder>label_ph</zorder>
  <zorder>groupBox</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_Q</zorder>
  <zorder>Digital_clock</zorder>
  <zorder>pushButton_note</zorder>
  <zorder>pushButton_Cal</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
